libbeat/docs/command-reference.asciidoc

```diff
@@ -82,31 +82,34 @@ endif::[]
 [options="header"]
 |=======================
 |Commands |
-ifeval::[("{beatname_lc}"=="functionbeat")]
+ifeval::["{beatname_lc}"=="functionbeat"]
 |<<deploy-command,`deploy`>> | {deploy-command-short-desc}.
 endif::[]
 |<<export-command,`export`>> |{export-command-short-desc}.
 |<<help-command,`help`>> |{help-command-short-desc}.
 |<<keystore-command,`keystore`>> |{keystore-command-short-desc}.
-ifeval::[("{beatname_lc}"=="functionbeat")]
+ifeval::["{beatname_lc}"=="functionbeat"]
 |<<package-command,`package`>> |{package-command-short-desc}.
 |<<remove-command,`remove`>> |{remove-command-short-desc}.
 endif::[]
-ifeval::[("{beatname_lc}"=="filebeat") or ("{beatname_lc}"=="metricbeat")]
+ifeval::["{beatname_lc}"=="filebeat"]
```

Comment:
Should this be `ifdef::has_modules_command[]`?

Reply:
yup! You're right. I created the attribute to deal with NOT conditions, but I can use it here, too. Wheeeee.